packages/next/src/server/node-environment-extensions/utils.tsx

```diff
@@ -44,8 +44,25 @@ export function io(expression: string, type: ApiType) {
                 'Unknown expression type in abortOnSynchronousPlatformIOAccess.'
               )
           }
+
           const errorWithStack = new Error(message)
 
+          if (
+            process.env.NODE_ENV !== 'production' &&
+            workUnitStore.captureOwnerStack
```

Comment:
Why would it be undefined in dev?

Reply:
It wouldn't. But the alternatives are invariant error or non-null assertion. All three options kinda suck. Can you think of a fourth one?

Reply:
They are the ones who reflect what's actually happening though. This should never be nullable in dev so we should error instead of silently ignoring the defect. Any "invariant" wouldn't give us more info than we need though so this should just be a non-null assertion.